.github/workflows/test.yml

```diff
@@ -3,10 +3,30 @@ name: Wagtail CI
 on:
   push:
     paths-ignore:
-    - 'docs/**'
+      - '.circleci/**'
+      - 'docs/**'
+      - '.readthedocs.yml'
+      - '.squash.yml'
+      - 'CHANGELOG.txt'
+      - 'CODE_OF_CONDUCT.md'
+      - 'CONTRIBUTORS.rst'
+      - 'LICENSE'
+      - 'README.md'
+      - 'SECURITY.md'
+      - 'SPONSORS.md'
   pull_request:
     paths-ignore:
-    - 'docs/**'
+      - '.circleci/**'
+      - 'docs/**'
+      - '.readthedocs.yml'
+      - '.squash.yml'
+      - 'CHANGELOG.txt'
+      - 'CODE_OF_CONDUCT.md'
+      - 'CONTRIBUTORS.rst'
+      - 'LICENSE'
+      - 'README.md'
+      - 'SECURITY.md'
+      - 'SPONSORS.md'
```

Comment:
It might be… but one could argue that it's more conservative to use a blocklist, so that if new paths that should be checked are added, someone doesn't have to know to update the allowlist.